application/data/standardisers/ethnicity_classification_finder_builder.py

```diff
@@ -0,0 +1,139 @@
+from application.data.standardisers.ethnicity_classification_finder import (
+    EthnicityStandardiser,
+    EthnicityClassificationCollection,
+    EthnicityClassification,
+    EthnicityClassificationDataItem,
+    EthnicityClassificationFinder,
+)
+from application.utils import get_bool
+
+
+class EthnicityClassificationFileColumn:
+    """
+    An enum that defines columns when loading an EthnicityClassificationFinder from a settings file
+    """
```

Comment:
It might be nice if these subclass `enum.Enum` so that they're legit Python enums?